src/site/content/en/blog/web-platform-03-2023/index.md

```diff
@@ -0,0 +1,124 @@
+
+{% BrowserCompat 'css.types.color_value.color-mix' %}
+
+The Chrome release also includes [new DevTools](https://developer.chrome.com/blog/new-in-devtools-111/#color) to help you work with this new color functionality.
+
+{% Aside %}
+You can also test the new color functions `color()`, `lab()`, `lch()`, `oklab()`, and `oklch()` in Firefox 111, currently behind a preference `layout.css.more_color_4.enabled`.
+{% endAside %}
+
+### More control over `:nth-child()` selections
+
+Chrome 111 adds the ability to pass a selector list into `:nth-child()` and `nth-last-child()`. Learn more about this, and see examples in the post [More control over :nth-child() selections with the of S syntax](https://developer.chrome.com/articles/css-nth-child-of-s/).
+
+{% BrowserCompat 'css.selectors.nth-child.of_syntax' %}
+
+### Support for previous and next slide in the Media Session API
+
+Finally in this list of Chrome 111 additions are the [Presenting slides actions](/media-session/#presenting-slides-actions) for the media session API—`"previousslide"` and `"nextslide"`.
+
+{% BrowserCompat 'api.MediaSession.nextslide_type' %}
+
+### Pseudo-class support in Safari
+
+Safari 16.4 is an amazing release for the web platform, this article won't cover all of the additions so check out the full list of features in the [Safari 16.4 release notes](https://developer.apple.com/documentation/safari-release-notes/safari-16_4-release-notes). 
+
+There's a bunch of additional CSS pseudo-classes with support in this release: `:user-invalid`, `:user-valid`, `:dir()`, `:modal`, and `:fullscreen`.
+
+### New range syntax for media queries
+
+This Safari release makes the far more elegant and useful range syntax for media queries interoperable across all three engines. See examples of this syntax in [this post](https://developer.chrome.com/blog/media-query-range-syntax/), published when the syntax shipped in Chrome. 
+
+{% BrowserCompat 'css.at-rules.media.range_syntax' %}
+
+### CSS Properties and Values
+
+Safari 16.4 adds support for [`@property`](https://developer.mozilla.org/docs/Web/CSS/@property) enabling CSS custom property registration directly in a stylesheet. Learn more about this in [@property: giving superpowers to CSS variables](/at-property/).
+
+{% BrowserCompat 'css.at-rules.property' %}
+
+### CSS API support
+
+The great additions for CSS keep coming, with support for the [CSS Typed OM](https://developer.mozilla.org/docs/Web/API/CSS_Typed_OM_API). This API exposes CSS values as typed JavaScript objects rather than strings. It makes working with CSS from JavaScript easier, and is more performant than existing methods.
```

Comment:
```suggestion
The great additions for CSS keep coming, with support for the [CSS Typed OM](https://developer.mozilla.org/docs/Web/API/CSS_Typed_OM_API). This API exposes CSS values as typed JavaScript objects rather than strings. It makes working with CSS from JavaScript easier, and outperforms existing methods.
```